context.go

```diff
@@ -1,4 +1,5 @@
 // Copyright 2014 The Macaron Authors
+// Copyright 2020 The Gitea Authors
```

Comment:
I do not think this is an appropriate addition unless this is a maintained fork version.

Disclaimer: I am not good at legal terms.

Reply:
```suggestion
```